<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabPage</class>
 <widget class="QMainWindow" name="TabPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">font-family:'Consolas';</string>
   </property>
   <widget class="QWidget" name="widget_python" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <property name="text">
          <string>Python解释器</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_pythonexe">
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_choosepython">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_choosepython{
border-radius:3%;
border:1px solid;
background:rgb(42.5, 28.8, 28.8);
border-color:rgb(132.5, 64, 64);
color:#F56C6C;
}

#pushButton_choosepython::hover{
background:#F56C6C;
border-color:#F56C6C;
color:white;
}

#pushButton_choosepython::pressed{
background:rgb(247, 137.4, 137.4);
border-color:rgb(247, 137.4, 137.4);
color:white;
}</string>
         </property>
         <property name="text">
          <string>选择解释器</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget_space_1" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>本地pip-第三方库</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView_pip">
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_shortcut" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>381</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="7" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_run_python">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="keySequence">
        <string>Ctrl+Shift+F10</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_openprogram">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="keySequence">
        <string>Ctrl+O</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_openprogram">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="text">
        <string>打开项目文件夹</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_left_change_file">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="keySequence">
        <string>Alt+Q</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_save_file">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="keySequence">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_save_all">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="keySequence">
        <string>Ctrl+Shift+S</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_new_file">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="text">
        <string>新建文件</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_run_python">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="text">
        <string>运行程序</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_save_file">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="text">
        <string>保存文件</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_new_file">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="keySequence">
        <string>Ctrl+N</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_close_file">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="text">
        <string>关闭文件</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_save_all">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="text">
        <string>保存全部文件</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_right_change_file">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="text">
        <string>右切换项目文件</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_right_change_file">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="keySequence">
        <string>Alt+E</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_close_file">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="keySequence">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_left_change_file">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="text">
        <string>左切换项目文件</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_clean_output">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="text">
        <string>清理当前输出</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QKeySequenceEdit" name="keySequenceEdit_clean_output">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
font-size:14px;
font-weight:700;</string>
       </property>
       <property name="keySequence">
        <string>F1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
